client/src/app/(metadraft)/(validator)/rules-selection/table/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { useRxCollection } from "rxdb-hooks";
+
+import { Label } from "~/components/ui/label";
+import { Switch } from "~/components/ui/switch";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "~/components/ui/table";
+
+import { camelCaseToTitleCase } from "~/lib/camel-case-to-title-case";
+import { type Rule, RULES_DESCRIPTION } from "~/lib/rules";
+import type { RulesCollection } from "~/lib/types";
+import { cn } from "~/lib/utils";
+
+export default function TableRules({
+  keys,
+  rules,
+}: {
+  keys: Rule[];
+  rules: RulesCollection;
+}) {
+  const rulesCollection = useRxCollection<RulesCollection>("rules");
+
+  const handleChange = async (checked: boolean, key: Rule) => {
+    try {
+      const newRules = { ...rules };
+      if (checked) {
+        newRules.rules.push(key);
+      } else {
+        newRules.rules = newRules.rules.filter((k) => k !== key);
+      }
+      await rulesCollection?.upsert(newRules);
+    } catch (e) {
+      console.error(e);
+    }
+  };
+
+  return (
+    <Table>
+      <TableHeader className="h-14 bg-secondary text-white/50 hover:bg-secondary [&>*]:border-white/30">
+        <TableRow>
+          <TableHead className="w-52">Rule Name</TableHead>
+          <TableHead>Definition</TableHead>
+          <TableHead className="w-20">Active</TableHead>
+        </TableRow>
+      </TableHeader>
+      <TableBody className="[&_tr:last-child]:border-1 [&>*]:border-white/30">
+        {keys.map((key) => (
+          <TableRow
+            key={key}
+            className={cn(rules.rules.includes(key) && "bg-success/10")}
```

Comment:
I don't the bg-success/10  is a little too much.
I would suggested making the selected rules normal (no special bg) and the unselected rules greyed out (bg-white/10 or something like that).